bin/cmsl1t_batch_resubmit

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+import click
+import click_log
+import glob
+import logging
+import pandas as pd
+import re
+import os
+import sys
+from tabulate import tabulate
+import shutil
+
+from cmsl1t.batch import Batch, Status, condor_resubmit
+from cmsl1t.batch.common import PATHS
+
+if sys.version_info[0] < 3:
+    from StringIO import StringIO
+else:
+    from io import StringIO
+
+logger = logging.getLogger(__name__)
+click_log.basic_config(logger)
+
+
+def resubmit_jobs(df, batch_dir):
+    config_files = list(df['config_file'])
+    config_files = [batch_dir + cfg for cfg in config_files]
+    local_ids = list(df['local_id'])
+    batch_log_dir = os.path.join(batch_dir, PATHS.LOG)
+    run_script = os.path.join(batch_dir, PATHS.RUN_SCRIPT)
+
+    results = condor_resubmit(config_files, local_ids, batch_dir,
+                              batch_log_dir, run_script)
+    new_df = pd.DataFrame(results)
+    new_df['local_id'] = local_ids
+    return new_df
+
+
+@click.command()
+@click.option(
+    '-i', '--info_file', help='path to info.csv', type=click.Path(exists=True),
```

Comment:
Is it nicer functionality to enable this command run the function to generate the info file itself, rather than need the user to run the status command and then the resubmit command in quick succession?  Would also help avoid a situation where a finished job is accidentally resubmitted because the info file was generated quite a while ago and the jobs status has changed (though I'm not sure if this is realistically ever going to happen).

Reply:
Perhaps you could even merge this and the `status` command into a `monitor` command and just have an option to switch on the auto-resubmit behaviour, otherwise it defaults to just printing the status. What do you think?

Reply:
The info file is created on submit in `bin/cmsl1t_batch`. Since you can run multiple "batches" of batch jobs, it is required to point the script to it. It could be the input folder, it could be the info file. I've chosen the latter here just for coding convenience.

Regarding auto-resubmit: Ganga has that functionality. I am planning to use that further down the line.